src/fp/file/fp_file.c

```diff
@@ -233,15 +246,20 @@ struct Menu *createFileMenu(void) {
                   "thunder rage\0",
                   menuByteOptionmodProc, &pm_gCurrentSaveFile.globalBytes[0xD9]);
     menuAddStatic(&menu, 0, y, "toy box 3", 0xC0C0C0);
-    struct MenuItem *toyboxItem3Option = menuAddOption(&menu, menuX, y++,
-                                                       "pokey\0"
-                                                       "koopatrol\0"
-                                                       "super soda\0",
-                                                       menuByteOptionmodProc, &pm_gCurrentSaveFile.globalBytes[0xDA]);
+    menuAddOption(&menu, menuX, y++,
+                  "pokey\0"
+                  "koopatrol\0"
+                  "super soda\0",
+                  menuByteOptionmodProc, &pm_gCurrentSaveFile.globalBytes[0xDA]);
+    menuAddStatic(&menu, 0, y, "audio mode", 0xC0C0C0);
+    struct MenuItem *audioOption = menuAddOption(&menu, menuX, y++,
+                                                 "mono\0"
+                                                 "stereo\0",
+                                                 audioModeProc, &pm_gSaveGlobals.useMonoSound);
```

Comment:
Looks like useMonoSound is only checked at [startup](https://github.com/pmret/papermario/blob/5929e49b7237271a1f4b2b62f29f5082cc3c30a8/src/state_startup.c#L75) and in the file menu, so I don't think this will have an effect without reloading the game. Don't know if that was your intention?